websites/A/Aniorb/presence.ts

```diff
@@ -0,0 +1,107 @@
+const presence = new Presence({
+  clientId: "776312991939428373"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+});
+let video = {
+video: false,
+duration: 0,
+currentTime: 0,
+paused: true
+};
+  const timestamps = presence.getTimestamps(
+    Math.floor(video.currentTime),
+    Math.floor(video.duration)
+  );

```

Comment:
*It's const*

Reply:
Wait its not suppose to be?

Reply:
See below, and tbh I don't see the point why you define it here